util/util.go

```diff
@@ -1006,3 +1006,16 @@ func VerifySnapshotLabels(labels map[string]string) error {
 func RemoveNewlines(input string) string {
 	return strings.Replace(input, "\n", "", -1)
 }
+
+type ResourceGetFunc func(kubeClient *clientset.Clientset, name, namespace string) (runtime.Object, error)
+
+func WaitForDeletion(kubeClient *clientset.Clientset, name, namespace, resource string, maxRetryForDeletion int, getFunc ResourceGetFunc) error {
```

Comment:
Can we rename it to `WaitForResourceDeletion`?

Reply:
Renamed. Thank you.